data/remnant/remnant 1 introduction.txt

```diff
@@ -50,7 +50,7 @@ mission "First Contact: Remnant"
 			`	Several of them gather around the one with the scanner as if waiting to see what the results will be. As you watch them you notice that all of them, even the women, have their hair cropped short. Also, their clothing looks like nothing that has ever been in fashion at any point in human history that you know of.`
 			`	Finally the scanner beeps, and a yellow light flashes. You can almost feel the tension leaving the air as the crowd relaxes. "So you are pure. You are human," chants one of the women. "You are one of us."`
 			choice
-				`	"Of course I'm human. But what are human beings doing out here?"`
+				`	"Of course I am human. But what are human beings doing out here?"`
```

Comment:
None of your reasoning seems to apply to the player's dialog at this point, or indeed at any point until they start using sign language exclusively.

Reply:
That particular one (and others that fall into the same time frame) are more for practical purposes: If someone is talking to someone who is not fluent in their language (or any other linguistic/communication problems), it is best practice to enunciate carefully and speak words fully without contractions. This is clearly such a situation, so it is appropriate for the character to do so.

On a similar note, this is a quasi first-contact / diplomatic communication. I think it is highly inappropriate to be excessively casual, as portrayed by all the slang.

Reply:
> This is clearly such a situation

Is it? There is no indication that they are having trouble speaking, only that they choose to chant. There is the suggestion that we aren't sure if they understand our initial response, but that seems more related to their expressions, or lack thereof, than an idea that we think they don't have a good grasp of the language they are asking if we comprehend. The further we go, the more they demonstrate that they do in fact grasp the human language.

Reply:
As far as that goes, if you pick the "just reply" option instead of trying to chant, you get:

> `	Their expressions are inscrutable; you are not sure if they even understood you or not. They turn away from you for a second and converse in sign language.`

That aside, it's still a bunch of strangers in an almost alien culture, it remains appropriate to speak with more formality than a casual chat as if we've been best friends for the last decade. These are people that will shoot you if you so much as scan them, and likewise shoot you if you try to not give them a blood sample. Being polite and saying the full words instead of being overly casual is a very reasonable thing to do.